### ISSUE TITLE
Update opens a validation tab (yes, again, but it's annoying)

### ISSUE THREAD

**Issue by L0rent** [Tampermonkey/tampermonkey#2259]
Hi. I would like to make script updates completely silent but I don't know how to do it. “Don't ask me for simple script updates” is checked. However some of my scripts are updated frequently, open a new tab, ask for manual update validation, and irritate me because these update requests disrupt my work. This script for example https://codeberg.org/Amm0ni4/bypass-all-shortlinks-debloated causes a new tab to open.
Post 904 talks about using @require. I asked where it should be placed, but nobody answered.
Violentmonkey updates scripts completely silently. I would like Tampermonkey to do the same.
Do you need more information?
Please help me.

TL;DR

## Expected Behavior

Scripts should update silently.

## Actual Behavior

Although the “Don't ask me for simple script updates” option is checked, updating certain scripts opens a tab asking me to validate the update. I'm aware that this may be more than a “simple” update, but I'd just like Tampermonkey to do the update silently.

## Specifications

- Firefox : 128.0
- TM: 5.3.2
- OS: Windows 10 Enterprise 22H2 19045.4412

## Script

Bypass All Shortlinks Debloated 

(https://codeberg.org/Amm0ni4/bypass-all-shortlinks-debloated/src/commit/0356f69da855c9484dba6fbb47a0181a8a1c1c16/Bypass_All_Shortlinks.user.js#bypass=true
